python/build.py

```diff
@@ -85,10 +85,13 @@ def get_python_executable():
     shutil.copy('../runtimes/RelWithDebInfo/taichi_core.dll',
                 'taichi/lib/taichi_core.pyd')
 
-os.system('{} -m taichi changelog --save && cat ../CHANGELOG.md'.format(
+os.system('cd .. && {} -m taichi changelog --save && cat CHANGELOG.md'.format(
     get_python_executable()))
```

Comment:
Actually, does this work on Windows?

Reply:
I'm not sure, I triggered `[release]` mode for [AppVeyor](https://ci.appveyor.com/project/yuanming-hu/taichi/builds/33487532) to test, hope this works.

Reply:
AppVeyor passes, so it should work.